newrelic/hooks/datastore_aioredis.py

```diff
@@ -21,17 +21,10 @@
     _redis_multipart_commands,
     _redis_operation_re,
 )
+from newrelic.common.package_version_utils import get_package_version_tuple
 
 
-def get_aioredis_version():
-    try:
-        import aioredis as aioredis_legacy
-    except ModuleNotFoundError:
-        return None
-    try:
-        return tuple(int(x) for x in getattr(aioredis_legacy, "__version__").split("."))
-    except Exception:
-        return 0, 0, 0
+get_aioredis_version = lambda: get_package_version_tuple("aioredis")
```

Comment:
I only see that this is used in one place. Instead of making this a lambda can we just replace its usage on line 66 with the following?
```
aioredis_version = get_package_version_tuple("aioredis")
```

Reply:
Will do